en_US/LC_MESSAGES/django.po

```diff
@@ -11506,364 +10988,1072 @@ msgstr ""
+msgid "Accept this Revision"
+msgstr "Accept this Revision"
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:25
+#, python-format
+msgid "You are about to approve your own edit. Please consider asking our <a href=\"%(url)s\">reviewers</a> instead."
+msgstr "You are about to approve your own edit. Please consider asking our <a href=\"%(url)s\">reviewers</a> instead."
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:32
+#, fuzzy
+msgid "Clicking Accept will make this version of the article live! Please take a moment to be sure all options are correct."
+msgstr "Clicking Accept will make this version of the article live! Please pick the appropriate revision level below and consider sending a thank you to the person who submitted this."
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:38
+msgid "Thanks for taking the time to review these changes. Clicking Accept will make this version of the article live! Please take a moment to be sure all options are correct."
+msgstr "Thanks for taking the time to review these changes. Clicking Accept will make this version of the article live! Please take a moment to be sure all options are correct."
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:71
+msgid "Accept"
+msgstr "Accept"
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:76
+#, fuzzy
+msgid "Defer this Translation"
+msgstr "Reject this Translation"
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:76
+#, fuzzy
+msgid "Defer this Revision"
+msgstr "Reject this Revision"
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:81
+#, fuzzy
+msgid ""
+"Thanks for taking the time to review these changes. Please thank the editor and then give specific, constructive feedback about what you feel needs to change in order for the revision to be "
+"acceptable."
+msgstr "Thanks for taking the time to review these changes. Please consider sending a thank you to the person who submitted them."
+
+#: kitsune/wiki/jinja2/wiki/includes/review_form.html:95
+msgid "Defer"
+msgstr "Defer"
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:6
+msgid "Show Helpfulness Votes Chart"
+msgstr "Show Helpfulness Votes Chart"
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:8 kitsune/wiki/jinja2/wiki/includes/revision_list.html:127
+msgid "Compare Selected Translations"
+msgstr "Compare Selected Translations"
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:8 kitsune/wiki/jinja2/wiki/includes/revision_list.html:127
+#, fuzzy
+msgid "Compare Selected Revisions"
+msgstr "Compare Selected Translations"
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:25
+msgid "Significance - MT: Major Content Changes. M: Content Changes. Empty: Minor Changes."
+msgstr "Significance - MT: Major Content Changes. M: Content Changes. Empty: Minor Changes."
+
+#: kitsune/wiki/jinja2/wiki/includes/revision_list.html:25
+msgctxt "First letter of 'Significance'"
+msgid "S"
+msgstr "S"
```

Comment:
Do you have space to accommodate the translation for Significance, non-Latin scripts will most likely translate it. Same with R in the following string